### ISSUE TITLE
Interstellar rockets

### ISSUE THREAD

**Issue by Ithronyar** [zmaster587/AdvancedRocketry#1144]
Minecraft Version: 1.12.2
Forge Version: 1.12.2-14.23.2.2611
Advanced Rocketry Version: 1.2.6-55
LibVulpes Version: 0.2.8-27
**is this the latest curseforge build?** Yes

I put a warp core on a rocket, tried to fly it to another star... and it didn't work. I assume this is intentional, judging by the message that popped up on the screen about only being able to travel within the same solar system. So I built the actual warp core multiblock into the rocket, put some dilithium crystals in.... and I still got the "you can't go there" popup.

That just doesn't make sense, so I'm going to consider it a bug or an oversight. Even just the warp core block on the rocket should be enough for interstellar travel, since rockets are far lighter and more mobile than space stations.

If it _is_ intentional, it's an annoying and counterintuitive limitation. I'm currently testing some custom systems, and I need to travel between stars. Since their planets have random IDs, I can't just warp myself there. And I don't have a space station because I'm regularly regenerating dimensions. Now I'm stuck and have to go to a space station and build a warp core, just to test a few variables for a star. Quite a waste of time.
Another example: Say a player dies in another solar system, and spawns back on Earth. Their space station is currently light-years away as well, but not to worry: They have a giant backup rocket built with a warp core just for this situation. They get in, try to launch, and... it doesn't work. Anger and despair presumably ensue after that.
In short, this is a problem. Rockets are more mobile than space stations, yet they can't travel between stars. It makes testing tedious, it doesn't make sense, and it can create a huge and arbitrary game reset if the player dies in another solar system and can't reach their space station.

**Comment by C1ffisme** [comment]
So, I'd like to bring up the question of whether or not this is balanced before anyone develops this feature. Although it would be pretty cool, there's a little bit more progression required to building a space station, and it does possibly limit us from expanding the requirements for any spacecraft to be interstellar.

However, one thing I would like to see are rockets that can get to other planets in the same solar system. Maybe a special rocket that, when you reach outer space, opens up a GUI where you can control the rocket using the arrow keys and basic Newtonian physics? I'll leave that idea for later.

I think, at least for now, the mod needs a manual to tell the player that this won't work so that he won't try it and be disappointed when it doesn't work.

**Comment by Ithronyar** [comment]
I don't think balance is an issue here; just require a whole warp core multiblock to be built into the rocket to travel between stars. Same cost as the space station. The single-block warp core would still work for interplanetary travel like it does now. And since you can't really bring along a whole facility full of machines and infrastructure, taking a rocket would still be inferior to warping with a space station.

**Comment by C1ffisme** [comment]
> And since you can't really bring along a whole facility full of machines and infrastructure, taking a rocket would still be inferior to warping with a space station.

I suppose that's true...

**Comment by dmodoomsirius** [comment]
Is this still a issue? Please comment if it is and i will reopen if if you are unable? We are closing old issues to see what is still relevant and needs to be fixed.

**Comment by Ithronyar** [comment]
> 
> 
> Is this still a issue? Please comment if it is and i will reopen if if you are unable? We are closing old issues to see what is still relevant and needs to be fixed.

Still an issue in v1.5.0-131.
![2019-03-05_00 52 13combined](https://user-images.githubusercontent.com/26017430/53793011-d98d7180-3ee1-11e9-94a8-8fe95028f609.png)

**Comment by ViciousGamer** [comment]
For now, until this is added, you can use these two commands to go to planets (that arent gas giants)

`/advancedRocketry planet list`

`/advancedRocketry goto [dimid]`

**Comment by zmaster587** [comment]
It is intentional that warp cores on rockets do not allow interstellar travel, the idea being that the warp core alone without dilithium fuel does not have the power to go to a nearby star.  I wanted to allow for another use for the core without requiring building a separate block for the rocket.
